storage/tokudb/ha_tokudb.cc

```diff
@@ -161,6 +161,15 @@ void TOKUDB_SHARE::static_init() {
 void TOKUDB_SHARE::static_destroy() {
     my_hash_free(&_open_tables);
 }
+const char* TOKUDB_SHARE::get_state_string(share_state_t state) {
```

Comment:
Coming back to this, actually, a static method is probably more appropriate for this. Will change and repush.